### ISSUE TITLE
run TF on SLURM cluster

### ISSUE THREAD

**Issue by gundramleifert** [gundramleifert/exp_tf#2]
Challenging task... We want to use TF on SLURM-Cluster - do distribute the TF graph over different nodes is the target.

**Comment by gundramleifert** [comment]
one link: http://stackoverflow.com/questions/34826736/running-tensorflow-on-a-slurm-cluster

**Comment by gundramleifert** [comment]
one more... https://github.com/tensorflow/tensorflow/issues/1686

**Comment by gundramleifert** [comment]
https://www.uibk.ac.at/zid/systeme/hpc-systeme/

**Comment by gundramleifert** [comment]
https://ischlag.github.io/2016/06/12/async-distributed-tensorflow/

**Comment by gundramleifert** [comment]
https://github.com/tensorflow/tensorflow/issues/3517

**Comment by ahundt** [comment]
@gundramleifert Have you been able to get this working?

**Comment by gundramleifert** [comment]
No, I am stucking in a very early point - just distribute on 3 cores it does not scale (only one is calculating). In addition, there are other issues with higher priority. I will inform you, if I have a process here...
